tracer/src/Datadog.Trace.BenchmarkDotNet/DatadogExporter.cs

```diff
@@ -110,6 +111,30 @@ public IEnumerable<string> ExportToFiles(Summary summary, ILogger consoleLogger)
                         testName += report.BenchmarkCase.Parameters.DisplayInfo;
                     }
 
+                    // The Job Id can contain random values: https://github.com/dotnet/BenchmarkDotNet/blob/ec429af22e3c03aedb4c1b813287ef330aed50a2/src/BenchmarkDotNet/Jobs/JobIdGenerator.cs#L18
+                    // Example:
+                    //      Job-WISXTD(Runtime=.NET Framework 4.7.2, Toolchain=net472, IterationTime=2.0000 s)
+                    // We use the description as part of the configuration facets.
+                    // The configuration facets are used by CI Visibility to create a fingerprint of the test.
+                    // Random values here means a new test fingerprint every time, so there's no way to track the same
+                    // test in multiple executions. So because of that, we need to remove the random part of the description.
+                    // In this case we remove the job name and keep the information between the parethesis.
+                    var description = report.BenchmarkCase.Job?.DisplayInfo;
+                    if (description is not null)
+                    {
+                        var match = new Regex(@".*\((.*)\)").Match(description);
+                        if (match.Success && match.Groups.Count == 2)
+                        {
+                            // In the example: Runtime=.NET Framework 4.7.2, Toolchain=net472, IterationTime=2.0000 s
+                            description = match.Groups[1].Value;
+                        }
+                        else if (description?.StartsWith("Job-") == true)
+                        {
+                            // If we cannot extract the description but we know there's a random Id, we prefer to skip the description value.
+                            description = string.Empty;
```

Comment:
Is there a difference between sending `null` (which it could be) or `string.Empty` here?